docs/ru/docs/tutorial/index.md

```diff
@@ -0,0 +1,80 @@
+# Учебник - Руководство пользователя - Введение
+
+В этом руководстве шаг за шагом показано, как использовать **FastApi** с большинством его функций.
+
+Каждый раздел постепенно основывается на предыдущих, но он структурирован по отдельным темам, так что вы можете перейти непосредственно к конкретной теме для решения ваших конкретных потребностей в API.
```

Comment:
```suggestion
.. так что вы можете напрямую/быстро перейти к интересующей теме для решения нужд вашего API.
```

`конкретно` is repeated twice, it's better to avoid it